sharktank/requirements.txt

```diff
@@ -5,7 +5,7 @@ gguf==0.10.0
 numpy<2.0
 
 # Needed for newer gguf versions (TODO: remove when gguf package includes this)
-# sentencepiece>=0.1.98,<=0.2.0
+sentencepiece>=0.1.98,<=0.2.0
```

Comment:
See also https://github.com/nod-ai/shark-ai/pull/676#discussion_r1880805172

Reply:
See response https://github.com/nod-ai/shark-ai/pull/676#discussion_r18826010700. v0.11.0 released an hour ago, which has `sentecepiece` included as a dep

Reply:
Upgrading `gguf` to `0.11.0` and removing `sentencepiece` dep in #676

Reply:
Ah, I saw the tag for `0.11.0` but there was no whl on PyPI and the `project.toml` looked identical. Thanks for updating!